python/bayesmixpy/run.py

```diff
@@ -44,27 +45,64 @@ def run_mcmc(
         hierarchy: str,
         mixing: str,
         data: np.array,
-        dens_grid: np.array,
         hier_params: str,
         mix_params: str,
         algo_params: str,
+        dens_grid: np.array = None,
         out_dir: str = None):
 
+    """
+    Run the MCMC sampling by calling the Bayesmix executable from a subprocess.
+    Arguments
+    ---------
+    hierarchy: the id of the hyerarchy. Must be one of ["NNIG", "NNW", "LinRegUni"]
+    mixing: the id of the mixing. Must be one of ["DP", "PY", "LogSB", "TruncSB"]
```

Comment:
Is it a good idea to write this, so that we have to go back and update it whenever a new object is added? Can't we just have an official list of implemented objects and their appropriate id, and reference that instead? Such a list would have several advantages IMO

Reply:
Completely right, I've pushed a branch called `document_protos`, which adds the proto messages to the docs. Could you publish this version of the documentation? (I forgot how to do it)
Then we just link to the page of hierarchy_id / mixing_id ecc...

Reply:
Do you mean that the RTFD documentation should be built off of the `document_protos` branch?

Reply:
Yes just for the moment. I am opening a PR right now, but it's very preliminary stuff. I just made so that a "proto" page gets generated along the way